portfolio/src/main/webapp/script.js

```diff
@@ -88,3 +88,22 @@ async function getDataAsyncAwait() {
   document.getElementById('data-container').innerText = data;
 }
 
+function getServerComments() {
+  fetch('/data').then(response => response.json()).then((comments) => {
+    // stats is an object, not a string, so we have to
```

Comment:
Comment isn't clear. Whats stats?

Reply:
It was in reference to the walk-throughs so I removed it.